python/mxnet/gluon/block.py

```diff
@@ -885,6 +885,50 @@ class SymbolBlock(HybridBlock):
     >>> x = mx.nd.random.normal(shape=(16, 3, 224, 224))
     >>> print(feat_model(x))
     """
+    @staticmethod
+    def import_(symbol_file, input_names, param_file=None, ctx=None):
```

Comment:
I don't think `import_` is a great name, even though I understand that we want to be as close as possible to `.export` and `import being a key-word makes it impossible to use `.import()`.
I would suggest adding an alias to export, `export_symbols` `export_to_symbol` `export_hybridized`, etc and have the matching one for import. That way it would be clearer, as adding a `_` suffix is not a conventionnal thing to do and is close to the use of `_` as a prefix, which is for private functions.